common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -188,3 +188,9 @@ def definition_to_xml(self, resource_fs):
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
         return xml_object
+
+    @property
+    def non_editable_metadata_fields(self):
+        non_editable_fields = super(SequenceDescriptor, self).non_editable_metadata_fields
+        non_editable_fields.extend([SequenceFields.is_entrance_exam, ])
```

Comment:
I actually wonder if "is_entrance_exam" should be Scope.content instead of Scope.settings. To ME it feels more like Scope.content. But this has always been a fuzzy area. @cpennington, what do you think?

If it is changed to Scope.content, you don't need the non_editable_metadata override. Some day we may allow editing of Scope.content elements in the "Settings editor", but we would have to deal with a number of new "non-editable" settings at that time.